contracts/teller-network/MetadataStore.sol

```diff
@@ -338,7 +336,7 @@ contract MetadataStore is Stakable, MessageSigned, SecuredFunctions {
     function offer(uint256 _id) external view returns (
         address asset,
         string memory currency,
-        int8 margin,
+        int16 margin,
```

Comment:
With this change we're allowing a margin % that goes from -32,768 to +32,767.  Do you think w should limit the max and min value in a condition in the offer creation function?

Reply:
You'd say it's better to do it in the contract, in the form or both?

Reply:
Hmm... in the smart contract it would be a nice to have but it's not super important, because it's only an informative value. I wonder what would happen if someone puts -101% and a price of 50usd for some crypto? Would the UI display a minimum limit of -0.5usd?

And in the case of the form, yeah, I think we should have some valid limits, starting from -99 to some upper limit %?